### DISCUSSION TITLE
CMake error while trying to run Halide

### DISCUSSION THREAD

**Discussion by ahmed-zubair-1998** [halide/Halide#5004] [General]
I have installed halide binaries for windows and I am trying to run bgu for some days. For this I am trying to run CMakeLists.txt inside apps/bgu folder. I have read README_cmake so I am running the following command inside the folder.
`cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_MODULE_PATH="/path/to/Halide" .`
But I am getting the following error.

> CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "FindHalide.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Halide", but
  CMake did not find one.
  Could not find a package configuration file provided by "Halide" with any
  of the following names:
    HalideConfig.cmake
    halide-config.cmake
  Add the installation prefix of "Halide" to CMAKE_PREFIX_PATH or set
  "Halide_DIR" to a directory containing one of the above files.  If "Halide"
  provides a separate development package or SDK, be sure it has been
  installed.
-- Configuring incomplete, errors occurred!

I tried to do this on ubuntu where I get the same error. This is my first time using cmake so I might be making some rookie mistake.

**Comment by benzwt** [comment]
Not familiar with cmake.
But I can build apps/bgu by "make", there is a makefile in the directory.

**Comment by alexreinking** [comment]
The binaries in the latest release do not provide a proper CMake build. Please build Halide from master and read `README_cmake.md`.
